app/src/views/app/AppInfo.vue

```diff
@@ -290,31 +338,15 @@ async function uninstall() {
         </template>
       </p>
 
-      <p>
-        <YIcon iname="comments" /> {{ $t('app.info.problem') }}
-        <a :href="`https://forum.yunohost.org/tag/${id}`" target="_blank">
-          {{ $t('app.info.forum') }}
-        </a>
-      </p>
-
-      <VueShowdown :markdown="app.description" />
+      <VueShowdown :markdown="app.description" class="px-2" />
```

Comment:
cosmetics, would have removed the padding
```suggestion
      <VueShowdown :markdown="app.description" />
```